js/models/listing/Metadata.js

```diff
@@ -0,0 +1,52 @@
+import BaseModel from '../BaseModel';
+import is from 'is_js';
+import { getCurrencyByCode } from '../../data/currencies';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      contractType: 'PHYSICAL_GOOD',
+      listingType: 'FIXED_PRICE', // this is not in the design at this time
+      // by default, setting to "never" expire (due to a unix bug, the max is before 2038)
+      expiry: (new Date(2037, 11, 31, 0, 0, 0, 0)).toISOString(),
+    };
+  }
+
+  get contractTypes() {
+    return [
+      'PHYSICAL_GOOD',
+      'DIGITAL_GOOD',
+      'SERVICE',
+      'CROWD_FUND',
+    ];
+  }
+
+  // todo: validate the listing type is one of the available types
```

Comment:
no, that one is for the contractType. Still need to do listingType validation at some point. I don't think the listing type is even in the designs at this point.